typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -31,9 +31,11 @@ const chainsToDeploy = [
   'swell',
 ];
 
-const MAX_PROTOCOL_FEE = parseEther('100').toString(); // Changing this will redeploy the PROTOCOL_FEE hook
+export const MAX_PROTOCOL_FEE = parseEther('100').toString(); // Changing this will redeploy the PROTOCOL_FEE hook
 
-const protocolFee = (0.5 / Number(tokenPrices.ethereum)).toFixed(10).toString(); // ~$0.50 USD
+export const protocolFee = (0.5 / Number(tokenPrices.ethereum))
```

Comment:
This is applied on all chains right? On ezETH we have some chains that don't use ETH as the gas token (e.g. BSC) and we probably wanna be robust to more in the future

so should this maybe be

```
export function getProtocolFee(chain: ChainName) {
  // same thing, but `tokenPrices[chain]`
```

Reply:
great idea